### DISCUSSION TITLE
Is full-example.rc fully safe for vanilla rtorrent?

### DISCUSSION THREAD

**Discussion by Rouneq** [kannibalox/pyrosimple#63] [Q&A] [answered]
Question comes from a comment at the top of full-example.rc. It says "less rTorrent-PS specific" which implies not completely rTorrent-PS independent. Not being versed in differences between vanilla and PS, what should I avoid/comment out when evaluating what to integrate with my current vanilla rc?

**Comment by kannibalox** [comment, marked as answer]
Yep! I haven't tried it in a while but I believe even the original pyrocore config is completely safe for vanilla, mostly I just went through and removed a lot of the conditional code that would only have been enabled on rTorrent-PS. `full-example.rc` is intended to show the full configuration required to integrate between vanilla and `pyrosimple`, although naturally it would also work with -PS just fine as well.
